### PR TITLE
Fixes: #128 Add support for adding bind to /etc/redis.conf

### LINKED ISSUE #128: RFE - control bind address with environment variable
I'm deploying software into OpenShift and would like to use this image as a sidecar container to run alongside my application.

I'd like to request an environment variable that I can set to cause redis to listen only on `127.0.0.1` instead of `0.0.0.0`.

Current workaround is to rebuild the container image with this `Dockerfile`:

```
FROM registry.redhat.io/rhel8/redis-6:latest

USER 0

RUN echo $'log_info "Configuring redis to listen on localhost only ..."\necho "bind 127.0.0.1" >> "${REDIS_CONF}"\n' > /usr/share/container-scripts/redis/post-init.sh

USER 1001
```

### PR DESCRIPTION
**Pull request by phracek**
Add support for adding bind to /etc/redis.conf
by BIND_ADDRESS variable

Closes #128

### CODE REVIEW

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test-openshift]

**Comment by phracek** [comment]
@yrro @pkubatrh @hhorak @zmiklank Please review it.

File: 6/root/usr/share/container-scripts/redis/README.md [hunk 1]
```diff
@@ -57,6 +57,9 @@ Environment variables and volumes
 **`REDIS_PASSWORD`**  
        Password for the server access
 
+**`BIND_ADDRESS`**
+       Configuration redis to listen on this address
```
Comment (yrro) [inline 6/root/usr/share/container-scripts/redis/README.md:61]: Typo - `Configuration` -> `Configure`

**Review by yrro — CHANGES_REQUESTED** [review]
LGTM thanks, just spotted a typo

**Review by yrro — CHANGES_REQUESTED** [review]

File: 7/root/usr/share/container-scripts/redis/README.md [hunk 2]
```diff
@@ -56,6 +56,9 @@ Environment variables and volumes
 **`REDIS_PASSWORD`**  
        Password for the server access
 
+**`BIND_ADDRESS`**
+       Configuration redis to listen on this address
```
[the remark this answers is not in the document]
Comment (phracek) [reply 7/root/usr/share/container-scripts/redis/README.md]: Fixed by https://github.com/sclorg/redis-container/pull/153/commits/8668c141a066d647df6597a41db1c3f710141ec4

File: 6/root/usr/share/container-scripts/redis/README.md (hunk 1 above)
Comment (phracek) [reply 6/root/usr/share/container-scripts/redis/README.md]: Fixed by https://github.com/sclorg/redis-container/pull/153/commits/8668c141a066d647df6597a41db1c3f710141ec4

**Comment by phracek** [comment]
[test]

**Pull request merged**